### ISSUE TITLE
Extra Colour Issue

### ISSUE THREAD

**Issue by GTMango** [itsjustcurtis/MenyooSP#56]
Having an issue revolving around car colors. I'm running this mod here: https://www.gta5-mods.com/misc/skysderworks-community-color-pack-skysder, alongside https://www.gta5-mods.com/misc/chameleon-paint-add-on. Not only do the Chameleon color section under Menyoo Customs show as "Extra Colour", but the colours of all other category fail to load either. I imagine this use of these two mods is the cause of this. Is there any way to work around this?
![20240125031949_1](https://github.com/itsjustcurtis/MenyooSP/assets/141836327/a5d1c5f0-3e5d-4357-b8ef-13b89657d662)
![20240125031945_1](https://github.com/itsjustcurtis/MenyooSP/assets/141836327/76e5fa44-8699-4dd7-9a23-1c520a423679)
![20240125031939_1](https://github.com/itsjustcurtis/MenyooSP/assets/141836327/edcae890-8be8-4e56-9a0b-f4cd26c06205)

**Comment by itsjustcurtis** [comment]
Hi, thanks for reporting. What you are experiencing is by design; If the game is unable to find a name for the colour, then Menyoo will list it as "Extra Colour x" I have both of these mods installed and working fine. Be sure you have installed them correctly as Rockstar recently updated the way that both chameleon and addon paint colours are loaded.

**Comment by GTMango** [comment]
Heard, I'll run through them once more then. Thanks again. Is there a better way to contact you about things like this so we're not opening issues for non-issues?

**Comment by itsjustcurtis** [comment]
I'm in the process of creating a discord server, just need to finish off a
few things and I'll link that on the mod page and here

**Comment by GTMango** [comment]
Hey man. I've been playing around with the addon colors mods to make sure I've got em installed right and turns out everything's been put in correctly. I've tried removing all scripts, reverting back to a stock update.rpf and even old Menyoo versions, but a lot of times, the colors don't work correctly. And this is for all colors. The only colors that do work right are the Utility, Matte and Worn colors. I'm still looking through to see what else can I do to fix this, but is there anything else that would affect the way Menyoo loads its colors?
